docs/flasharray-nvmeof.md

```diff
@@ -0,0 +1,37 @@
+# FlashArray NVMeoF Support
+
+## Introduction
+
+The Pure Service Orchestrator Kubernetes CSI driver includes support for FlashArray NVMeoF RDMA SAN type
+starting from PSO v6.2.0, explicitly NVMeoF-RoCE.
+
+## Prerequisites
+
+* PSO v6.2.0 or higher
+* FlashArray Purity 5.2.0 or higher
+* nvme-cli 1.8.1 or higher
+* Mellanox OFED 2.3 or higher
```

Comment:
Do we need `Mellanox` requirement?  Do NVMe team has a doc to specify the requirement for FA-NVMe-RoCE? We can add the reference here.